plothist/plotters.py

```diff
@@ -308,6 +310,7 @@ def plot_2d_hist_with_projections(
         ax_colorbar=ax_colorbar,
         pcolormesh_kwargs=pcolormesh_kwargs,
         colorbar_kwargs=colorbar_kwargs,
+        square_ax=False,
```

Comment:
What about `square_ax=figsize[0]==figsize[1]`?

Reply:
if `figsize[0]==figsize[1]`, this subplot is square shape by construction or the subplots placement, no need.

Reply:
OK